### PR TITLE
release-->main

### PR DESCRIPTION
**Pull request by Raj-StepSecurity**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/step-security/setup-gh-cli-action.git"
+  },
+  "keywords": [
+    "CLI",
+    "GitHub"
+  ],
+  "author": "ser.soft GmbH",
```
Comment (varunsh-coder) [inline package.json:25]: same here

Comment (Raj-StepSecurity) [reply package.json]: done

File: action.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+name: Setup GitHub CLI
+description: Installs or updates the GitHub CLI (gh)
+author: ser.soft GmbH
```
[the remark this answers is not in the document]
Comment (Raj-StepSecurity) [reply action.yml]: done

File: .github/workflows/tag-update.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    types: [ published ]
+
+jobs:
+  update-tags:
+    if: ${{ !github.event.release.prerelease }}
+    name: Update Running Releases
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: step-security/running-release-tags-action@v3
```
[the remark this answers is not in the document]
Comment (Raj-StepSecurity) [reply .github/workflows/tag-update.yml]: no, my bad, removed

**Review by varunsh-coder — APPROVED** [review]

**Pull request merged**